### PR TITLE
DevTools 108 > CSS hints + new doc on CSS issues

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-108/#css-hint

Also, created a new doc that explains how the Styles pane shows invalid, overridden, inactive, inherited, shorthand, and non-editable CSS.

Publish on Nov 29.

### CODE REVIEW

File: site/en/docs/devtools/css/issues/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+The **Styles** pane crosses out properties with invalid syntax and displays {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hJxGejygnlBfMvhMtcCX.svg", alt="Warning.", width="24", height="24" %} warning icons next to them.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/HSQATBEOt6MholSjoji2.png", alt="Invalid CSS properties.", width="800", height="542" %}
+
+### Overridden {: #overridden }
+
+The **Styles** pane crosses out properties that are overridden by other properties according to the [Cascading order][3].
+
+{% Video src="video/NJdAV9UgKuN8AhoaPBquL7giZQo1/BNlYQHj8tpojjxyMKNDv.mp4", autoplay=false, controls=true, loop=true, class="screenshot" %}
```
Comment (jecfish) [inline site/en/docs/devtools/css/issues/index.md:38]: How about skip the navigation, or explain it with an image?

File: site/en/docs/devtools/css/issues/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+### Inactive {: #inactive }
+
+The **Styles** pane crosses out properties that are valid but have no effect because of other properties. Such properties have {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2OyGRodvgk9neTIEAH4g.svg", alt="Information.", width="24", height="24" %} information icons next to them and are inactive because of CSS logic, not the [Cascading order][3].
```
Comment (jecfish) [inline site/en/docs/devtools/css/issues/index.md:42]: Which properties do `Such properties` referring to? Is that the cross-out prop or the pale prop? Suggest to make the sentence to call them out clearer.

Suggest to explain the example.

**Review by jecfish — CHANGES_REQUESTED** [review]
Thanks for this write-up! You gave me idea on the upcoming CSS tip.

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/css/reference/index.md [hunk 3]
```diff
@@ -44,6 +44,12 @@ There are many ways to select an element:
 
 ## View CSS {: #view }
 
+### View invalid, overridden, inactive, and other CSS {: #css-issues }
```
Comment (jecfish) [inline site/en/docs/devtools/css/reference/index.md:47]: Looking at the CSS class names, some classes are grey out as well. Should we explain it in this docs?

ScreenFlow

**Review by jecfish — APPROVED** [review]
LGTM overall % a few comments

**Pull request merged**